assets/app/scripts/app.js

```diff
@@ -110,11 +110,34 @@ angular
       }) 
       .when('/project/:project/browse/services', {
         templateUrl: 'views/services.html'
-      })       
+      })
+
+      .when('/oauth', {
+      	templateUrl: 'views/util/oauth.html',
+        controller: 'OAuthController'
+      })
+      .when('/error', {
+      	templateUrl: 'views/util/error.html',
+        controller: 'ErrorController'
+      })
+      .when('/logout', {
+      	templateUrl: 'views/util/logout.html',
+        controller: 'LogoutController'
+      })
+
       .otherwise({
         redirectTo: '/'
       });
   })
+  .config(function(AuthServiceProvider, RedirectLoginServiceProvider) {
+  	AuthServiceProvider.setLoginService('RedirectLoginService');
+  	AuthServiceProvider.setUserStore('SessionUserStore');
```

Comment:
Probably a (yet to be created) cookie based one. Otherwise you have to log in every refresh. 



> On Feb 3, 2015, at 8:42 PM, Jessica Forrester <notifications@github.com> wrote:
> 
> In assets/app/scripts/app.js:
> 
> >        .otherwise({
> >          redirectTo: '/'
> >        });
> >    })
> > +  .config(function(AuthServiceProvider, RedirectLoginServiceProvider) {
> > +  	AuthServiceProvider.setLoginService('RedirectLoginService');
> > +  	AuthServiceProvider.setUserStore('SessionUserStore');
> should we fall back to MemoryUserStore when sessionStorage isn't available?
> 
> —
> Reply to this email directly or view it on GitHub.
>